cli/command/cli.go

```diff
@@ -169,20 +168,6 @@ func (cli *DockerCli) ContentTrustEnabled() bool {
 	return cli.contentTrust
 }
 
-// BuildKitEnabled returns whether buildkit is enabled either through a daemon setting
-// or otherwise the client-side DOCKER_BUILDKIT environment variable
-func BuildKitEnabled(si ServerInfo) (bool, error) {
```

Comment:
docker/compose relies on this to select the builder to be used.
Maybe restore this function as a dumb `return true` would make more sense?

Reply:
Looks like the logic was moved inside `processBuilder()`. Perhaps it makes sense to move that logic back into this function. The default would now be (`true`) if `DOCKER_BUILDKIT` is not set?

Wondering if the function also needs to take `windows` daemon into account 🤔 
How exactly is it used in Compose (as in; what does it do if it's "disabled"? or is that only for the Windows case?)